miso-web/src/main/webapp/scripts/hot_boxable.js

```diff
@@ -90,6 +90,19 @@ HotTarget.boxable = (function() {
             },
             pack: function(obj, flat, errorHandler) {
               // search field only
+            },
+            depends: ['discarded', 'distributed'],
+            update: function(obj, flat, flatProperty, value, setReadOnly, setOptions, setData) {
+              var applyChanges = function(source, autoSelect) {
+                setOptions({
+                  source: source
+                });
+                setData(source.length > 1 && !autoSelect ? 'SELECT' : source[0]);
+              };
+              if ((flatProperty === 'discarded' && value == 'True') || (flatProperty == 'distributed' && value == 'Sent Out')) {
+                setData(null);
+                setReadOnly(true);
+              }
```

Comment:
Need `else` block to call `setReadOnly(false);`

Box Alias column below should depend on `discarded` similarly to this column

Reply:
Done